client/gutenberg/extensions/subscriptions/edit.jsx

```diff
@@ -0,0 +1,69 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { Component } from '@wordpress/element';
+import { TextControl, FormToggle } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import { __ } from 'gutenberg/extensions/presets/jetpack/utils/i18n';
+import { sprintf } from '@wordpress/i18n/build/index';
+
+class SubscriptionEdit extends Component {
+	render() {
+		const { attributes, className, isSelected, instanceId, setAttributes } = this.props;
+		const {
+			subscribe_placeholder,
+			subscribe_button,
+			success_message,
+			show_subscribers_total,
+		} = attributes;
+
+		if ( isSelected ) {
+			return (
+				<div className={ className }>
+					{ __( 'Show total subscribers' ) }
+					<FormToggle
```

Comment:
This seems like an interesting mix of preview and controls. I'd expect this toggle to be in the `InspectorControls` instead, and not part of the block preview.

Reply:
The `FormToggle` component seems to be designed to be only a sidebar component, which is probably why it's design is broken